perf/lua/CMakeLists.txt

```diff
@@ -23,6 +23,8 @@ function(create_perf_lua_test)
                     COMMAND env
                       LUA_PATH="${LUA_PATH}"
                       ${TARANTOOL_BIN} ${TEST_PATH}
+                        --output="${PERF_OUTPUT_DIR}/${PERF_NAME}.json"
+                        --output_format=json
```

Comment:
Raw idea: make the options verbatim follow the `gbench` ones. This way we can add more useful options from `gbench` step-by-step. And possibly end with a convenient benchmarking microframework with some compatibility with `gbench` for integrations of various kinds.

Reply:
I've thought about it and prefer to set a different flags intentionally. The status of the `gbench` isn't very clear for me.
For example, google/benchmark#794 reports that CSV format is depricated and will be dropped soon, but still there is no progress. Also, I am not sure how we want to match `--benchmark_min_time`, `--benchmark_min_warmup_time`, `--benchmark_repetitions` with our Lua benchmarks and how they are useful. Plus, this `benchmark` prefix in flags looks obsolete to me.

But I'm OK to rename flags to match the `gbench` ones if you insist.

Reply:
OK, I've no objection. Just checked the idea with you.